enrichment-common/src/main/java/eu/europeana/enrichment/common/commons/HelperFunctions.java

```diff
@@ -69,6 +75,45 @@ public static String createHttpRequest(String content, String baseUrl) {
 			return "";
 		}
 	}
+	
+	public static String createHttpRequestSparkTopicDetection(String content, String baseUrl, String nrTopics) {

```

Comment:
This class is more suitable for more general functions like createHttpRequest. Anyway, I have already implemented the topic detection service as you worked on the topic apis, as we agreed in the meeting, you did not need to implement this.